packages/big-design/src/utils/messagingHelpers.tsx

```diff
@@ -25,6 +27,10 @@ export type MessageLinkItem = Pick<LinkProps, 'external' | 'href' | 'target'> &
   text: string;
 };
 
+export interface MessageAction extends Omit<ButtonProps, 'children'> {
```

Comment:
Do you think we should limit the Button's `variant` prop to `secondary` and `subtle`? There should only be 1 `primary` button on a page, and I would assume it wouldn't be in a messaging component.

cc. @valterfatia @eugene-polev

Reply:
I think it makes sense

Reply:
Guys, please let me know if everyone agrees with the proposal from @chanceaclark and I will make changes to the current PR 🙇

Reply:
@chanceaclark @eugene-polev 
I'm done with this topic so, now available only 2 variants: `secondary` and `subtle`(docs & screenshots were updated).